src/org/zaproxy/zap/extension/params/ParamsPanel.java

```diff
@@ -204,23 +204,26 @@ private JScrollPane getJScrollPane() {
 	
 	private void setParamsTableColumnSizes() {
 		
-		paramsTable.getColumnModel().getColumn(0).setMinWidth(50);
-		paramsTable.getColumnModel().getColumn(0).setPreferredWidth(100);	// type
+		paramsTable.getColumnModel().getColumn(0).setMinWidth(60);
+		paramsTable.getColumnModel().getColumn(0).setPreferredWidth(60);	// type
 		
 		paramsTable.getColumnModel().getColumn(1).setMinWidth(100);
-		paramsTable.getColumnModel().getColumn(1).setPreferredWidth(200);	// name
+		paramsTable.getColumnModel().getColumn(1).setPreferredWidth(100);	// name
 		
-		paramsTable.getColumnModel().getColumn(2).setMinWidth(50);
-		paramsTable.getColumnModel().getColumn(2).setPreferredWidth(100);	// used
+		paramsTable.getColumnModel().getColumn(2).setMinWidth(40);
+		paramsTable.getColumnModel().getColumn(2).setPreferredWidth(40);	// used

```

Comment:
This column is now trimmed. For example, in French this column and "% Change" are trimmed as well.

May I suggest removing the hardcoded sizes and pack the columns? Didn't try it but based on other tables it should look fine in more languages, although we would have to take care to not use the whole value for the last column, given how bigger the values are.